tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/PacketTracer.cs

```diff
@@ -0,0 +1,260 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System.Runtime.Intrinsics.X86;
+using static System.Runtime.Intrinsics.X86.Avx;
+using static System.Runtime.Intrinsics.X86.Avx2;
+using static System.Runtime.Intrinsics.X86.Sse2;
+using System.Runtime.Intrinsics;
+using System;
+
+using ColorPacket256 = VectorPacket256;
+
+internal class Packet256Tracer
+{
+    public int Width { get; private set; }
+    public int Height { get; private set; }
+    private static readonly int MaxDepth = 5;
+
+    private static readonly Vector256<float> SevenToZero = SetVector256(7f, 6f, 5f, 4f, 3f, 2f, 1f, 0f);
+
+    public Packet256Tracer(int _width, int _height)
+    {
+        if ((_width % VectorPacket256.Packet256Size) != 0)
+        {
+            _width += VectorPacket256.Packet256Size - _width % VectorPacket256.Packet256Size;
```

Comment:
nit: My preference is to add operators where they help improve readability or where they help clarify operator precedence.